csharp/src/Microsoft.ML.OnnxRuntime/NativeMethods.cs

```diff
@@ -177,6 +177,10 @@ public struct OrtApi
         public IntPtr TensorAt;
         public IntPtr CreateAndRegisterAllocator;
         public IntPtr SetLanguageProjection;
+        public IntPtr SessionGetProfilingStartTimeNs;
+        public IntPtr SetGlobalIntraOpNumThreads;
+        public IntPtr SetGlobalInterOpNumThreads;
+        public IntPtr SetGlobalSpinControl;
```

Comment:
To sync the same order as `onnxruntime_c_api.cc`: add `SetGlobalIntraOpNumThreads`, `SetGlobalInterOpNumThreads`, `SetGlobalSpinControl` between `SessionGetProfilingStartTimeNs` and `AddInitializer`.

Reply:
1) The order must be the same as in c_api.cc. 2) You can't insert anything in betweemn when adding the new API. Append only, otherwise it will break backwards compatibility.

---
In reply to: [495642137](https://github.com/microsoft/onnxruntime/pull/5280#discussion_r495642137) [](ancestors = 495642137)